jobs/datadog-route-emitter/templates/http_check.yaml.erb

```diff
@@ -0,0 +1,11 @@
+init_config: {}
+
+instances:
+  - name: route_emitter debug endpoint
+    url: http://localhost:<%= p('diego.route_emitter.debug_addr').split(':')[1] %>/debug/pprof/cmdline
```

Comment:
Why pass *address:port* in `diego.route_emitter.debug_addr` when we only use the port and hard code the address? Is it because the `diego.route_emitter.debug_addr` property is a pre-existing property used for something else?